bin/probot-run.js

```diff
@@ -11,10 +11,10 @@ program
   .usage('[options] <apps...>')
   .option('-a, --app <id>', 'ID of the GitHub App', process.env.APP_ID)
```

Comment:
My personal preference for this would be to move the GitHub app args to be the second set of args, and let the -p, -t, -w to appear first in the list. But that was a lot of changes to introduce in the first piece of this PR, especially with the negative impact on readability of the PR.

This would look something like:
```
  .usage('[options] <apps...>')
  .option('-p, --port <n>', 'Port to start the server on', process.env.PORT || 3000)
  .option('-t, --tunnel <subdomain>', 'Expose your local bot to the internet', process.env.SUBDOMAIN || process.env.NODE_ENV !== 'production')
  .option('-w, --webhook-path <path>', 'URL path which receives webhooks. Ex: `/webhook`', process.env.WEBHOOK_PATH)
  .option('-a, --app <id>', 'ID of the GitHub App', process.env.APP_ID)
  .option('-s, --secret <secret>', 'Webhook secret of the GitHub App', process.env.WEBHOOK_SECRET)
  .option('-P, --private-key <file>', 'Path to certificate of the GitHub App', findPrivateKey)
 .parse(process.argv)
```


Let me know if others are :+1: on the switching of the group order.

Reply:
:+1: looks good. I like that its dev-stuff, then real-stuff.